networks: Ability to enable/disable without JS

SunilMohanAdapa opened the pull request:
- Avoid using a drop down menu.

- Separate out the status indicator and activate/deactivate button.

- Closes #173.

fonfon commented:
Merged the changes and made some tiny adaptions:
- Increased the displayed column size from col-sm-6 to col-sm-8 (so I don't get any linebreaks with my display settings)
- Added a TODO: I couldn't get the details of one of my connections. That might be because of my VM setup: I switched on the FreedomBox before setting up the network correctly (which also caused #227 ). Nevertheless the situation of not getting a correct `interface_name` is not handled in the source code.

SunilMohanAdapa commented:
On 10/14/2015 06:31 PM, fonfon wrote:
> Merged the changes and made some tiny adaptions:
> 
>   * Increased the displayed column size from col-sm-6 to col-sm-8 (so I
>     don't get any linebreaks with my display settings)

I've changed this to col-lg-8, so that it would get even more space on
on small and medium layouts.  This is in the network diagram pull request.

>   * Added a TODO: I couldn't get the details of one of my connections.
>     That might be because of my VM setup: I switched on the FreedomBox
>     before setting up the network correctly (which also caused #227
>     <https://github.com/freedombox/Plinth/issues/227> ). Nevertheless
>     the situation of not getting a correct |interface_name| is not
>     handled in the source code.

I kind of resolved this by not showing device information when device is
not available.  Other information is still shown properly.  This is also
in the network diagram pull request.

-- 
Sunil
